### DISCUSSION TITLE
Can't reproduce the effect show on the page

### DISCUSSION THREAD

**Discussion by taozuhong** [primefaces/community#401] [PrimeVue]
I followed the guide to install `primevue` library, then use the component in my app, it could show on the page, but can't reproduce the effect show on the official website page, then I drop it.

**Comment by tugcekucukoglu** [comment]
I don't understand exactly what you mean by 'effect show on the page'. Which component did you use?

**Reply by taozuhong** [reply]
I like the UI effect, so I started to try it, then I tried the `InputText` component in my project.
Case 1: InputText with FloatLabel
Case 2: InputText with HelpText

As you see, I dropped it.
